UnitySDK/Assets/ML-Agents/Scripts/RewardProvider/LowLevelRewardProvider.cs

```diff
@@ -0,0 +1,77 @@
+
+namespace MLAgents.RewardProvider
+{
+    /// <summary>
+    /// A legacy reward provider that can be used in an Agent as a way to easily upgrade
+    /// from the old reward system.
+    /// </summary>
+    public class LowLevelRewardProvider : IRewardProvider
```

Comment:
I don't feel like `LowLevelRewardProvider` is the right word here. I would rather have something like `CumulativeRewardProvider` or `ManualRewardProvider` since the user has to add AddReward on it.

Reply:
I'll see if I can come up with a better name

Reply:
I went with CumulativeRewardProvider